var/spack/repos/builtin/packages/blis/package.py

```diff
@@ -0,0 +1,69 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Blis(Package):
```

Comment:
Can you convert this to `AutotoolsPackage`? Everything looks pretty standard, you should only need to override `configure_args`.

Reply:
It started life in the spack universe as an `Autotools` package but realizing that they don't use `autotools`, I changed it back ! 

Browsing through the repo it looks like this is not designed as an `autotools` package so do we want to make it one here ? (see https://github.com/flame/blis/issues/17, https://github.com/flame/blis/issues/197, https://github.com/flame/blis/issues/195, etc )

Reply:
Ah, so this is a "fake" Autotools package that writes their own script that just happens to be called `configure`. In that case, you're right, this shouldn't be an Autotools package. Can you add a comment mentioning this so others aren't confused?